astropy/units/deprecated.py

```diff
@@ -41,6 +47,7 @@ def _initialize_module():
 # This generates a docstring for this module that describes all of the
 # standard units defined here.
 from .utils import generate_unit_summary as _generate_unit_summary  # noqa
+from .utils import generate_prefixonly_unit_summary as _generate_prefixonly_unit_summary  # noqa
```

Comment:
Just following what that module is already doing.  I know some of this stuff is full of circular-import landminds, so I'm guessing that's the reason?  And it's not really performance-sensitive so it only gets run once at import-time, practically speaking.